### PR TITLE
RStudio Team Rebrand

### PR DESCRIPTION
**Pull request by kellobri**
@colearendt

### CODE REVIEW

File: _site/index.html [hunk 1]
```diff
@@ -11,7 +11,7 @@
 
 
 
-<title>RStudio QuickStart - Getting Started</title>
```
Comment (colearendt) [inline _site/index.html:14]: @kellobri Do you know where this comes from? We probably prefer a more descriptive title

Comment (kellobri) [reply _site/index.html:14]: Should we change it to something like "Log in Links"? or "Landing Page" or ...ideas?

Comment (colearendt) [reply _site/index.html:14]: If you have any ideas, it would be helpful! Something better than `index.utf8` though 😄

File: _site/index.html [hunk 2]
```diff
@@ -11,7 +11,7 @@
...
+<title>index.utf8</title>
```
Comment (colearendt) [inline _site/index.html:14]: Sorry, this is the one that needs love.

**Review by colearendt — APPROVED** [review]

**Pull request merged**